dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/StatsTaskListGenerator.java

```diff
@@ -30,9 +33,12 @@ class StatsTaskListGenerator {
   ImmutableList<Task<?>> createTasks(ConnectorArguments arguments) throws IOException {
     ImmutableList<OracleStatsQuery> queries =
         ImmutableList.of(
-            OracleStatsQuery.create("hist-cmd-types", StatsSource.STATSPACK),
-            OracleStatsQuery.create("app-schemas-pdbs", StatsSource.METADATA),
-            OracleStatsQuery.create("app-schemas-summary", StatsSource.METADATA)
+            OracleStatsQuery.create("db-features", METADATA),
```

Comment:
I would suggest renaming `METADATA` to `NATIVE`.

Reply:
Done, I renamed the enum value

Reply:
I realized that the same change needs to be made to the subfolder in `resources`, I'll add it soon

Reply:
Feel free to prepare the change in a separate PR.